src/functionalTest/groovy/mpern/sap/commerce/build/BootstrapTest.groovy

```diff
@@ -156,7 +156,10 @@ class BootstrapTest extends Specification {
 
         when: "running bootstrap task"
 
-        def beforeBootstrap = Instant.now()
+        def beforeBootstrap = Instant.now() - 5
```

Comment:
i set the time to 5s earlier - this way we prevent the milisecond issue - we could also try to strip and and do an is after or equals check. the question is what is more desirable

Reply:
Lowering the precision to seconds + after equals is a bit cleaner

Reply:
done